tools/webhook-ingestion-pipeline/webhook_endpoint/consts.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2020 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Environment Variables
+PROJECT_ID = "GOOGLE_CLOUD_PROJECT"
+PUBSUB_TOPIC = "TOPIC"
+
+# Default Values
+MAX_CONTENT_SIZE = 1000000 * 5 # 5 megabytes
```

Comment:
nit avoid comments that could be improved var names.
```suggestion
MAX_CONTENT_MB = 1000000 * 5
```